src/SFA.DAS.EmployerIncentives.Commands/ApprenticeshipIncentive/RefreshLearner/LearnerMatchService.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Threading.Tasks;
+using SFA.DAS.EmployerIncentives.Commands.Persistence;
+using SFA.DAS.EmployerIncentives.Commands.Services.LearnerMatchApi;
+using SFA.DAS.EmployerIncentives.Domain.ApprenticeshipIncentives;
+using SFA.DAS.EmployerIncentives.Domain.Interfaces;
+
+namespace SFA.DAS.EmployerIncentives.Commands.ApprenticeshipIncentive.RefreshLearner
+{
+    internal class LearnerMatchService
+    {
+        private readonly ILearnerService _learnerService;
```

Comment:
The ILearnerService is actually the LearnerSubmission service since it deals with LearnerSubmission data